pkgs/applications/networking/cluster/terraform-providers/libvirt/default.nix

```diff
@@ -19,15 +19,23 @@
 
 buildGoPackage rec {
   pname = "terraform-provider-libvirt";
-  version = "0.6.2";
+  version = "0.6.3";
 
   goPackagePath = "github.com/dmacvicar/terraform-provider-libvirt";
 
+  patches = [
+    (fetchpatch {
```

Comment:
To fix the eval error `fetchpatch` has to be brought into scope.

Reply:
It strange why your [cherry-pick](https://github.com/NixOS/nixpkgs/pull/110964/commits/397228ca42dddd7b4a40a2176acf76e5baeb673a) of https://github.com/NixOS/nixpkgs/commit/bbdad5fa19532d596a55feaf28021f55e856c89c did not bring `fetchpatch` into scope because the latter does include it. Anyway, I would just add a new commit that fixes it.

Reply:
I messed up the cherry-pick trying to avoid picking changes from a6ce00c50c36624fec06b2b756a766d4d0f4a888. Missed the `fetchpatch` addition. Fixed now.